vendor/github.com/hlandau/passlib/README.md

```diff
@@ -0,0 +1,95 @@
+passlib for go
```

Comment:
That file (and a bunch of other files from gopkg.in/hlandau package) are deleted when I run the install-deps.
We should have the same result, that's troublesome!